CHANGELOG.md

```diff
@@ -82,14 +82,15 @@ and this project's packages adheres to [Semantic Versioning](http://semver.org/s
 
 - `cert-manager` upstream helm chart `v0.9.0`. ([#1](https://github.com/giantswarm/cert-manager-app/pull/1))
 
-[Unreleased]: https://github.com/giantswarm/cert-manager-app/compare/v2.0.0...master
-[v2.0.0]: https://github.com/giantswarm/cert-manager-app/releases/tag/v2.0.0
-[v1.0.8]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.8
-[v1.0.7]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.7
-[v1.0.6]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.6
-[v1.0.5]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.5
-[v1.0.4]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.4
-[v1.0.3]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.3
-[v1.0.2]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.2
-[v1.0.1]: https://github.com/giantswarm/cert-manager-app/releases/tag/v1.0.1
+[Unreleased]: https://github.com/giantswarm/cert-manager-app/compare/v2.0.1...HEAD
+[v2.0.1]: https://github.com/giantswarm/cert-manager-app/releases/tag/v2.0.0...v2.0.1
```

Comment:
```suggestion
[Unreleased]: https://github.com/giantswarm/cert-manager-app/compare/v2.0.0...HEAD
```

There's no 2.0.1 yet https://github.com/giantswarm/cert-manager-app/releases

Reply:
is this more architect magic then? it updates that itself?

Reply:
Yes, it will replace unreleased as part of the (pre-)release workflow